website/docs/docs/contributing/adapter-development/6-verifying-a-new-adapter.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Verifying a new adapter"
+id: "6-verifying-a-new-adapter"
+---
+
+## Why verify an adapter?
+
+Circa 2017,  dbt's adapter plugin framework was introduced to meet client demand back when dbt Labs (née Fishtown Analytics) still a data consultancy. As a result dbt Labs specialized in four data platforms and became experts in them. However, the covering the surface area of all possible databases, their respective nuances, and keeping them up-to-date and bug-free is a Herculean and/or Sisyphean task that couldn't be done by a single person or even a single team! Enter the dbt community which enables dbt Core to work on more than 30 different databases (32 as of Sep '22)!
+
+Free and open-source tools for the data professional are increasingly abundant. This is by-and-large a *good thing*, however it requires due dilligence that wasn't required in a paid-license, closed-source software world. Before taking a dependency on an open-source projet is is important to determine the answer to the following questions:
+
+1. Does it work?
+2. Does it meet my team's specific use case?
+3. Does anyone "own" the code, or is anyone liable for ensuring it works?
+4. Do bugs get fixed quickly?
+5. Does it stay up-to-date with new Core features?
+6. Is the usage substantial enough to self-sustain?
+7. What risks do I take on by taking a dependency on this library?
+
+These are valid, important questions to answer, especially given that the main package, dbt-core has only been issueing stable releases since October '21. Indeed, up until now, the majority of new user questions in database-specific channels are some form of:
+- "How mature is `dbt-<ADAPTER>`? Any gotchas I should be aware of before I start exploring?"
+- "has anyone here used `dbt-<ADAPTER>` for production models?"
+- "I've been playing with  `dbt-<ADAPTER>` -- I was able to install and run my initial experiments. I noticed that there are certain features mentioned on the documentation that are marked as 'not ok' or 'not tested'. What are the risks?
+I'd love to make a statement on my team to adopt DBT, but I'm pretty sure questions will be asked around the possible limitations of the adapter or if there are other companies out there using DBT with Oracle DB in production, etc."
```

Comment:
Are these direct quotes? Including the uppercase use of DBT? :)

I'd recommend either lowercasing, or `DBT [sic]` hah